pkg/logqlmodel/stats/context.go

```diff
@@ -52,6 +53,15 @@ type Context struct {
 	mtx sync.Mutex
 }
 
+type CacheType string
+
+const (
+	ChunkCache       CacheType = "chunk"
+	IndexCache                 = "index"
+	ResultCache                = "result"
+	WriteDedupeCache           = "write-dedupe"
```

Comment:
I've added the write dedupe cache here for the sake of completeness, but they are not yet displayed.